example_crud.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+import etesync as api
+
+
+class EtesyncCRUD:
+    def __init__(self, email, userPassword, remoteUrl, uid, authToken,
+                 cipher_key=None):
+        r"""Initialize EtesyncInterface
```

Comment:
The "r" here isn't needed, it denotes regular expression, which is not what this string is.
(same for the same thing in the rest of the file)

Reply:
OK